src/test/resources/tpcds/spark-2.4/approved-plans-v1_4/q1/explain.txt

```diff
@@ -1,255 +1,43 @@
+      :     :     +- BroadcastExchange HashedRelationBroadcastMode(List(input[1, bigint, true]))
+      :     :        +- *(6) Filter isnotnull((CAST(avg(ctr_total_return) AS DECIMAL(21,6)) * CAST(1.2 AS DECIMAL(21,6)))#8)
+      :     :           +- *(6) HashAggregate(keys=[ctr_store_sk#4], functions=[avg(ctr_total_return#7)])
+      :     :              +- Exchange hashpartitioning(ctr_store_sk#4, 5)
+      :     :                 +- *(5) HashAggregate(keys=[ctr_store_sk#4], functions=[partial_avg(ctr_total_return#7)])
+      :     :                    +- *(5) HashAggregate(keys=[sr_customer_sk#9, sr_store_sk#10], functions=[sum(UnscaledValue(sr_return_amt#11))])
+      :     :                       +- Exchange hashpartitioning(sr_customer_sk#9, sr_store_sk#10, 5)
+      :     :                          +- *(4) HashAggregate(keys=[sr_customer_sk#9, sr_store_sk#10], functions=[partial_sum(UnscaledValue(sr_return_amt#11))])
+      :     :                             +- *(4) Project [sr_customer_sk#9, sr_store_sk#10, sr_return_amt#11]
+      :     :                                +- *(4) BroadcastHashJoin [sr_returned_date_sk#12], [cast(d_date_sk#13 as bigint)], Inner, BuildRight
+      :     :                                   :- *(4) Project [sr_returned_date_sk#12, sr_customer_sk#9, sr_store_sk#10, sr_return_amt#11]
+      :     :                                   :  +- *(4) Filter (isnotnull(sr_returned_date_sk#12) && isnotnull(sr_store_sk#10))
+      :     :                                   :     +- *(4) FileScan parquet default.store_returns[sr_returned_date_sk#12,sr_customer_sk#9,sr_store_sk#10,sr_return_amt#11] Batched: true, Format: Parquet, Location: InMemoryFileIndex[file:/C:/Users/apdave/github/hyperspace-1/spark-warehouse/store_returns], PartitionFilters: [], PushedFilters: [IsNotNull(sr_returned_date_sk), IsNotNull(sr_store_sk)], ReadSchema: struct<sr_returned_date_sk:bigint,sr_customer_sk:bigint,sr_store_sk:bigint,sr_return_amt:decimal(...
+      :     :                                   +- ReusedExchange [d_date_sk#13], BroadcastExchange HashedRelationBroadcastMode(List(cast(input[0, int, true] as bigint)))
+      :     +- BroadcastExchange HashedRelationBroadcastMode(List(cast(input[0, int, true] as bigint)))
+      :        +- *(7) Project [s_store_sk#5]
+      :           +- *(7) Filter ((isnotnull(s_state#15) && (s_state#15 = TN)) && isnotnull(s_store_sk#5))
+      :              +- *(7) FileScan parquet default.store[s_store_sk#5,s_state#15] Batched: true, Format: Parquet, Location: InMemoryFileIndex[file:/C:/Users/apdave/github/hyperspace-1/spark-warehouse/store], PartitionFilters: [], PushedFilters: [IsNotNull(s_state), EqualTo(s_state,TN), IsNotNull(s_store_sk)], ReadSchema: struct<s_store_sk:int,s_state:string>
```

Comment:
I didn't check the comparison code, but is this `file:/C:/Users/apdave/github/hyperspace-1/spark-warehouse/store` normalized? I see the following in the original file: `Location [not included in comparison]/{warehouse_dir}/store_returns]`

Reply:
thanks @imback82 , I updated the normalize logic to work for hyperspace. Please take a look

Reply:
Sidenote: This error did not cause build failures because:

<query>/explain.txt is not used for validation. <query>/simplified.txt is used for comparison and throws error in case of failure.
In failure scenarios, the stderr output prints out both explain.txt and simplified.txt for user's benefit.